server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/api/SessionExtensionPoint.java

```diff
@@ -20,18 +20,17 @@
 
 import org.pf4j.ExtensionPoint;
 
+import com.oceanbase.odc.core.sql.execute.SessionOperations;
+
 /**
  * @author yaobin
  * @date 2023-04-14
  * @since 4.2.0
  */
-public interface SessionExtensionPoint extends ExtensionPoint {
-
-    void killQuery(Connection connection, String connectionId);
+public interface SessionExtensionPoint extends ExtensionPoint, SessionOperations {
```

Comment:
Why abstract a `SessionOperations` interface?

Reply:
we need to use `SessionExtensionPoint` at `odc-core` and we define `SessionExtensionPoint` at `connect-plugin-api`. 

so, we need to make `odc-core` rely on `connect-plugin-api` if we want to use `SessionExtensionPoint` at `odc-core` but `connect-plugin-api` reply on `odc-core` actually. That is to say that we can not make `odc-core` rely on `connect-plugin-api` to avoid circular dependency.

I create the `SessionOperations` and move some methods that located in `SessionExtensionPoint` originally and used by `odc-core` to `odc-core` to fix this problem.